### DISCUSSION TITLE
Add Full Support for System Transport Media Controls (GSTMC)

### DISCUSSION THREAD

**Discussion by Samuel12321** [ciderapp/Cider#881] [Feature Request]
Please add Add Full support for windows audio management (STMC) including:

[SystemMediaTransportControls](https://docs.microsoft.com/en-us/uwp/api/windows.media.systemmediatransportcontrols.updatetimelineproperties?view=winrt-19041)

[SystemMediaTransportControlsSessionManager​](https://docs.microsoft.com/en-us/uwp/api/windows.media.systemmediatransportcontrols?view=winrt-19041)

[SystemMediaTransportControls.PlaybackPositionChangeRequested](https://docs.microsoft.com/en-us/uwp/api/windows.media.systemmediatransportcontrols.playbackpositionchangerequested?view=winrt-19041)

[SystemMediaTransportControls.UpdateTimelineProperties](https://docs.microsoft.com/en-us/uwp/api/windows.media.systemmediatransportcontrols.updatetimelineproperties?view=winrt-19041)

**Comment by tva2000hn** [comment]
Windows NodeRT is a pain

**Reply by kengwang** [reply]
https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API
